spec/controllers/items_controller_spec.rb

```diff
@@ -103,11 +74,13 @@
     it 'should call dor-services to close the version' do
       expect(@item).to receive(:close_version)
       version_metadata = double(Dor::VersionMetadataDS)
-      allow(version_metadata).to receive(:current_version_id).and_return(2)
       allow(@item).to receive(:versionMetadata).and_return(version_metadata)
+      allow(version_metadata).to receive(:current_version_id).and_return(2)
+      allow(version_metadata).to receive(:tag_for_version).with("2")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.